src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -1249,12 +1249,12 @@ public IOpDelegationStrategy GetOpDelegationStrategy(UnaryOp op)
             return new DefaultUnaryOpDelegationStrategy(op, this);
         }
 
-        public ICallNodeDelegatableNodeValidationStrategy GetCallNodeDelegationStrategy()
+        public virtual ICallNodeDelegatableNodeValidationStrategy GetCallNodeDelegationStrategy()
```

Comment:
How is this different than
[https://github.com/microsoft/Power-Fx/pull/1170](https://github.com/microsoft/Power-Fx/pull/1170)

I already signed off on that one.